.mergify.yml

```
@@ -0,0 +1,11 @@
+queue_rules:
+ - name: default
+   conditions: []
+
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - base=main
+    actions:
+      queue:
+        name: default
```

I every once in a while saw invalid labels in merged pull requests. We can also remove some labels as a guard:
```suggestion
    actions:
      label:
        remove:
          - s:in-progress
          - s:review-needed
      queue:
        name: default
```

---

can't that be done by github rules?

---

Not that I know. If you can reference documentation about it I prefer to skip it, because I am in favor of setting all possible rules in the branch protection rules.